scripts/foundry/Utils.s.sol

```diff
@@ -307,83 +353,90 @@ contract Utils is Script, CommonUtils {
         }
         SafeTransaction[] memory transactions = new SafeTransaction[](calldatas.length);
         for (uint256 i = 0; i < calldatas.length; i++) {
-            transactions[i] = SafeTransaction(calldatas[i], targets[i], values[i], chainIds[i], operations[i], safes[i]);
+            transactions[i] = SafeTransaction(
+                calldatas[i],
+                targets[i],
+                values[i],
+                chainIds[i],
+                operations[i],
+                safes[i]
+            );
         }
         return transactions;
     }
 
     function _chainToMultiSend(uint256 chain) internal pure returns (MultiSend) {
-        if (chain == CHAIN_ETHEREUM) return multiSendEthereum;
-        else if (chain == CHAIN_POLYGON) return multiSendPolygon;
-        else if (chain == CHAIN_ARBITRUM) return multiSendArbitrum;
-        else if (chain == CHAIN_OPTIMISM) return multiSendOptimism;
-        else if (chain == CHAIN_AVALANCHE) return multiSendAvalanche;
-        else if (chain == CHAIN_GNOSIS) return multiSendGnosis;
-        else if (chain == CHAIN_BNB) return multiSendBNB;
-        else if (chain == CHAIN_POLYGONZKEVM) return multiSendPolygonZkEVM;
-        else if (chain == CHAIN_BASE) return multiSendBase;
-        else if (chain == CHAIN_CELO) return multiSendCelo;
-        else if (chain == CHAIN_LINEA) return multiSendLinea;
-        else if (chain == CHAIN_MANTLE) return multiSendMantle;
-        else if (chain == CHAIN_MODE) return multiSendMode;
-        else if (chain == CHAIN_BLAST) return multiSendBlast;
-        else if (chain == CHAIN_XLAYER) return multiSendXLayer;
+        if (chain == Constants.CHAIN_ETHEREUM) return multiSendEthereum;
```

Comment:
Because of the changes made in utils [PR#17](https://github.com/AngleProtocol/utils/pull/27/files). It's a better practice imo to explicitly import contracts, libs like this `import {CommonUtils} from "../src/CommonUtils.sol"` rather than `import from "../src/CommonUtils.sol"`. It avoids pollution and unexpected behaviours, for example when you are import two files that have similar variables names.